pkg/orchestrator/config/config.go

```diff
@@ -157,3 +158,23 @@ func extractOrchestratorDDUrl() (*url.URL, error) {
 	}
 	return URL, nil
 }
+
+// NewOrchestratorForwarder returns an orchestratorForwarder
+// if the feature is activated on the cluster-agent/cluster-check runner, nil otherwise
+func NewOrchestratorForwarder(confPath string, isCoreAgent bool) *forwarder.DefaultForwarder {
```

Comment:
You can simplify this function by calling `flavor.GetFlavor()` instead of having the argument `isCoreAgent`.
https://github.com/DataDog/datadog-agent/blob/master/pkg/util/flavor/flavor.go#L41